config/initializers/prx_auth.rb

```diff
@@ -0,0 +1,4 @@
+PrxAuth::Rails.configure do |config|
+  config.install_middleware = ENV['ID_HOST'].blank?
```

Comment:
Wait, what?  You install middleware only if the ID_HOST wasn't set?

Reply:
yep! the install_middleware flag only needs to be disabled if we are installing our own manually (https://github.com/PRX/cms.prx.org/pull/558/files#diff-b1fe55db50c712fef0673345e5b9c0d9R61 here) - but I also have a ticket in the PrxAuth::Rails gem to allow configuring those options directly through the rails config object so will circle back to that later. https://github.com/PRX/prx_auth-rails/issues/5